book/src/report.md

```diff
@@ -0,0 +1,18 @@
+# Generating a report
+
+Optopodi is geared to generate **reports**. These are CSV files that summarize important information about your repository or repositories. To create a report you need to take the following steps:
+
+- Make a directory `$DIR` for the report. We recommend a name like `data/2021-06-26`.
+- Create a `report.toml` file in that directory. You can start with [the template](https://github.com/optopodi/optopodi/blob/main/report-template.toml) and customize it.
```

Comment:
broken link for template at https://github.com/optopodi/optopodi/blob/main/report-template.toml

Reply:
ah maybe this won't be broken after merge... I see. You're two steps ahead 😉

Reply:
yes =)